TAScheduler/UnitTests/tests.py

```diff
@@ -141,40 +150,239 @@ class TestInstructorLecTAAsmgt(TestCase):
+            modality='In-person',
+            credits=3
+        )
+        self.tempCourse = CourseObj(self.hold_course)
+        self.user = User.objects.create(
+            email_address='user@example.com',
+            password='password',
+            first_name='Regular',
+            last_name='User',
+            home_address='123 User St',
+            phone_number='1234567893'
+        )
+
+    def test_remove_assignment(self):
+        self.tempCourse.removeAssignment(self.user)
+        # Check if the assignment (user) was removed from the course
+        assignments = self.tempCourse.getAsgmtsForCrse()
+        self.assertNotIn(self.user, assignments, "User was not removed from the course assignments")
 
 
 class TestCourseRemoveCourse(TestCase):  # Randall
-    pass
+    def setUp(self):
+        self.hold_course = Course.objects.create(
+            course_id=104,
+            semester='Winter 2025',
+            name='Basic Testing',
+            description='Basic course on testing.',
+            num_of_sections=1,
+            modality='Remote',
+            credits=2
+        )
+        self.tempCourse = CourseObj(self.hold_course)
+
+    def test_remove_course(self):
+        self.tempCourse.removeCourse()
+        # Check if the course was removed
+        course_exists = Course.objects.filter(id=self.hold_course.id).exists()
+        self.assertFalse(course_exists, "Course was not removed")
 
 
 class TestCourseEditCourseInfo(TestCase):  # Randall
-    pass
+    def setUp(self):
+        self.hold_course = Course.objects.create(
+            course_id=105,
+            semester='Fall 2025',
+            name='Testing Fundamentals',
+            description='A course on testing fundamentals.',
+            num_of_sections=4,
+            modality='Mixed',
+            credits=3
+        )
+        self.tempCourse = CourseObj(self.hold_course)
+        self.new_info = {"semester": "Spring 2026", "name": "Advanced Testing Fundamentals", "description": "new",
+                         "num_of_section": 3, "modality": "Online", "credits": 6}
+
+    def test_edit_course_info(self):
+        self.tempCourse.editCourse(self.new_info)
+        self.hold_course.refresh_from_db()
+        self.assertEqual(self.hold_course.semester, self.new_info["semester"], "Course semester was not updated")
+        self.assertEqual(self.hold_course.name, self.new_info["name"], "Course name was not updated")
+        self.assertEqual(self.hold_course.description, self.new_info["description"],
+                         "Course description was not updated")
+        self.assertEqual(self.hold_course.num_of_sections, self.new_info["num_of_sections"],
+                         "Course section was not updated")
+        self.assertEqual(self.hold_course.modality, self.new_info["modality"], "Course modality was not updated")
+        self.assertEqual(self.hold_course.credits, self.new_info["credits"], "Course credits was not updated")
```

Comment:
Have a test where some of the input does not match the required database formatting ie an int instead of a string